lib/rubocop/formatter/simple_text_formatter.rb

```diff
@@ -97,6 +87,55 @@ def message(offense)
         message = offense.corrected? ? green('[Corrected] ') : ''
         message << annotate_message(offense.message)
       end
+
+      # A helper class for building the report summary text.
+      class ReportSummarizer
+        include Colorizable, TextUtil
+
+        def initialize(file_count, offense_count, correction_count, context)
```

Comment:
I am not sure that we need to initialize instances of this class. I think it could be refactored to have static methods.